### ISSUE TITLE
Use new 'everchanged' operator instead of changed_from 1970

### ISSUE THREAD

**Issue by marco-c** [mozilla/relman-auto-nag#610]

**Comment by marco-c** [comment]
This is now usable.

**Comment by Snehaaa18** [comment]
can I solve this issue?

**Comment by suhaibmujahid** [comment]
> can I solve this issue?

@Snehaaa18 Yes, feel free to work on any issue as alon is it is not assigned or linked to an open PR.

**Comment by FACELESS-GOD** [comment]
@suhaibmujahid is Someone still  working on it or can I Solve this issue ??

**Comment by suhaibmujahid** [comment]
> is Someone still working on it or can I Solve this issue ??

@FACELESS-GOD you can work on it since it is not linked to an open PR.

**Comment by priyakumari02** [comment]
Hi @suhaibmujahid , Can I work on this issue also?

**Comment by suhaibmujahid** [comment]
>  Can I work on this issue also?

@priyakumari02 yes, same as https://github.com/mozilla/bugbot/issues/610#issuecomment-1874008080. Thank you for your interest.

**Comment by rmpasswd** [comment]
> > can I solve this issue?
> 
> @Snehaaa18 Yes, feel free to work on any issue as long as it is not assigned or linked to an open PR.

https://bugzilla.mozilla.org/show_bug.cgi?id=1546624#:~:text=5%20years%20ago-,Merged%20to%20master.,-Status%3A%20ASSIGNED%20%E2%86%92%20RESOLVED

according to bugzilla,  a pull request was merged, is this issue resolved? even though it says "Open".
